src/routes/signup.tsx

```
@@ -1,8 +1,10 @@
 import { createFileRoute, useNavigate, useRouterState } from '@tanstack/react-router';
-import { useEffect, useState } from 'react';
+import { FormEvent, useEffect, useState } from 'react';
```

ESLint

[error] 'FormEvent' is defined but never used.

(@typescript-eslint/no-unused-vars)